### PR TITLE
Update recipe: + bump patch version, ∆ change deps

### PR DESCRIPTION
**Pull request by mmcguffi**
Various improvements:
 - relax / fix deps
 - rm deprecated APIs
 - add tests
 - add CI
 - linting

### CODE REVIEW

File: recipes/plannotate/meta.yaml [hunk 1]
```diff
@@ -1,48 +1,46 @@
-{% set version = "1.2.0" %}
+{% set version = "1.2.1" %}
 
 package:
   name: plannotate
   version: {{ version }}
 
 build:
-  number: 4
+  number: 1
```
Comment (bgruening) [inline recipes/plannotate/meta.yaml:8]: should be reset to 0

File: recipes/plannotate/meta.yaml [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
+    - bokeh=2.4.1
 
 test:
   commands:
     - plannotate batch --help
 
 about:
-  home: https://github.com/barricklab/pLannotate
+  home: https://github.com/mmcguffi/pLannotate
   license: GPL-3
```
Comment (bgruening) [inline recipes/plannotate/meta.yaml:44]: can you please use here SPDX identifiers?

File: recipes/plannotate/meta.yaml (hunk 1 above)
Comment (mmcguffi) [reply recipes/plannotate/meta.yaml:8]: Sorry, thank you -- I released I forgot to change the sha256  and I am pointing to the wrong loc for the db download as well

File: recipes/plannotate/meta.yaml (hunk 2 above)
Comment (mmcguffi) [reply recipes/plannotate/meta.yaml:44]: Im not sure I understand: 
```
license: GPL-3
license_file: LICENSE
```
These are already in the meta.yam?

**Comment by mmcguffi** [comment]
@BiocondaBot please fetch artifacts

**Comment by mmcguffi** [comment]
@BiocondaBot please add label

**Comment by mmcguffi** [comment]
@BiocondaBot please merge

**Comment by mmcguffi** [comment]
@rpetit3 Im sorry to bother you, but could you review and merge this when you get a chance? Thank you! :)

**Review by rpetit3 — APPROVED** [review]

**Pull request merged**